input/fsh/GGGenomicStudy.fsh

```diff
@@ -130,7 +140,7 @@ Description:   "Genomic Study Analysis Output"
 * extension[file].value[x] only Reference(GenomicDataFile)
 * extension[type] ^short = "GenomicStudy.analysis.output.type"
 * extension[type].value[x] only CodeableConcept
-* extension[type].valueCodeableConcept from GenomicStudyDataFormatVS (example)
+* extension[type].valueCodeableConcept from GenomicStudyDataFormatVS (preferred)
```

Comment:
Simliar to line 100, should we add some example codes to ^short attribute to this data format?